### ISSUE TITLE
Logging in setup.py causes error in labextension develop

### ISSUE THREAD

**Issue by croesus** [jupyter-widgets/widget-cookiecutter#100]
There are log lines in setup.py (that are not in the widget-ts-cookiecutter setup.py):

```
log.set_verbosity(log.DEBUG)
log.info('setup.py entered')
log.info('$PATH=%s' % os.environ['PATH'])
```

The labextension install script relies on `setup.py --name` returning the package name. The log "setup.py entered" causes the following error:

```
widget % jupyter labextension develop . --overwrite
/xxx/widget/setup.py:42: DeprecatedWarning: create_cmdclass is deprecated as of 0.8 and will be removed in 1.0. "
Use `wrap_installers` to handle prebuild steps in cmdclass.
Use `get_data_files` to handle data files.
Use `include_package_data=True` and `MANIFEST.in` for package data.

  cmdclass = create_cmdclass('jsdeps', data_files_spec=data_files_spec)
/xxx/widget/setup.py:44: DeprecatedWarning: install_npm is deprecated as of 0.8 and will be removed in 1.0. Use `npm_builder` and `wrap_installers`
  install_npm(js_dir, npm=['yarn'], build_cmd='build:prod'), ensure_targets(jstargets),
An error occurred.
pkg_resources.extern.packaging.requirements.InvalidRequirement: Parse error at "'entered'": Expected stringEnd
See the log file for details:  /var/folders/z8/8x57p38d3c16c4x8gvz65b8w0000gn/T/jupyterlab-debug-xgjwrhhm.log

widget % cat /var/folders/z8/8x57p38d3c16c4x8gvz65b8w0000gn/T/jupyterlab-debug-xgjwrhhm.log

Traceback (most recent call last):

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/_vendor/packaging/requirements.py", line 98, in __init__
    req = REQUIREMENT.parseString(requirement_string)

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/_vendor/pyparsing.py", line 1654, in parseString
    raise exc

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/_vendor/pyparsing.py", line 1644, in parseString
    loc, tokens = self._parse( instring, 0 )

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/_vendor/pyparsing.py", line 1402, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/_vendor/pyparsing.py", line 3417, in parseImpl
    loc, exprtokens = e._parse( instring, loc, doActions )

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/_vendor/pyparsing.py", line 1406, in _parseNoCache
    loc,tokens = self.parseImpl( instring, preloc, doActions )

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/_vendor/pyparsing.py", line 3205, in parseImpl
    raise ParseException(instring, loc, self.errmsg, self)

pkg_resources._vendor.pyparsing.ParseException: Expected stringEnd (at char 9), (line:1, col:10)


During handling of the above exception, another exception occurred:


Traceback (most recent call last):

  File "/usr/local/lib/python3.9/site-packages/jupyterlab/debuglog.py", line 48, in debug_logging
    yield

  File "/usr/local/lib/python3.9/site-packages/jupyterlab/labextensions.py", line 135, in start
    ans = self.run_task()

  File "/usr/local/lib/python3.9/site-packages/jupyterlab/labextensions.py", line 206, in run_task
    develop_labextension_py(arg, user=self.user, sys_prefix=self.sys_prefix, labextensions_dir=self.labextensions_dir, logger=self.log, overwrite=self.overwrite,

  File "/usr/local/lib/python3.9/site-packages/jupyterlab/federated_labextensions.py", line 146, in develop_labextension_py
    m, labexts = _get_labextension_metadata(module)

  File "/usr/local/lib/python3.9/site-packages/jupyterlab/federated_labextensions.py", line 392, in _get_labextension_metadata
    pkg_resources.get_distribution(package)

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/__init__.py", line 469, in get_distribution
    dist = Requirement.parse(dist)

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3144, in parse
    req, = parse_requirements(s)

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3089, in parse_requirements
    yield Requirement(line)

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/__init__.py", line 3099, in __init__
    super(Requirement, self).__init__(requirement_string)

  File "/usr/local/lib/python3.9/site-packages/pkg_resources/_vendor/packaging/requirements.py", line 100, in __init__
    raise InvalidRequirement(

pkg_resources.extern.packaging.requirements.InvalidRequirement: Parse error at "'entered'": Expected stringEnd

Exiting application: lab
```

Deleting the logging code allows the installation to complete.

**Comment by maartenbreddels** [comment]
Those lines should be removed I think. I've had that in many libraries we maintain, and had to remove them all. I think if you open a PR, it will be merged soon.

**Comment by croesus** [comment]
Thanks @maartenbreddels for the feedback. I've just submitted the [PR](https://github.com/jupyter-widgets/widget-cookiecutter/pull/102). I wasn't confident that this was the correct fix when I raised the issue.

**Comment by jasongrout** [comment]
Fixed in #102.
